week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -24,21 +22,23 @@ function greaterThan10() {
 
 function sortArray() {
   let letters = ["a", "n", "c", "e", "z", "f"];
-  let sortedLetters;
-
+  let sortedLetters = letters.sort().toString();
```

Comment:
What does running `toString()` on an array do?

Reply:
Without it test will fail as arrays are not able to compare

Reply:
I see. It would be better to write the test turning it into a string then, and leave these functions to be more accurate.

[This is an example](https://github.com/CodeYourFuture/js-exercises/pull/469/files#r394408489) - not your fault, the tests just needed to be written better.